src/components/pages/mainPage/leftSideBar/accessBar/AccessPresentation.js

```diff
@@ -0,0 +1,38 @@
+import React, { Component } from 'react';
+import { Jumbotron, Button } from 'react-bootstrap';
+import Login from '../loginBar';
+import Register from '../registerBar';
+
+export default class AccessProfile extends Component {
+  constructor(props){
+    super(props);
+    this.state = {
+      register: false
+    }
+  }
+
+  register(e) {
```

Comment:
what is this method supposed to do?
should it be named something else?

Reply:
the component toggles between the registration and login forms

Reply:
In that case, it should be named something more descriptive like toggleRegisterForm and the state that contains the boolean should be something like showRegisterForm